### ISSUE TITLE
set maxBrightness

### ISSUE THREAD

**Issue by PrimaveraNL** [0x5e/homebridge-tuya-platform#6]
I have a dimmer in my system.
In the config JSON, I don't see the individual devices and therefore I have no means of setting maxBrightness for this device.
Do I overlook something?

**Comment by 0x5e** [comment]
This plugin pulls all of your devices in you account, currently no advance options for each device. Where did the `maxBrightness` came from ? I didn't see it in Tuya App and HomeKit.

**Comment by PrimaveraNL** [comment]
I found it in another plugin that I tried, it is possible to put that in the device config (JSON config) of that plugin.

**Comment by 0x5e** [comment]
What's the exactly meaning of `maxBrightness`? What behavior it is?
I'm afarid advance options may not in the high prority. When plugins became stable, and most of the device types are supported in basic functions, I will consider.
If anyone want's to implement this, welcome to create the PR :)

**Comment by PrimaveraNL** [comment]
Pls see the instruction set attached.
What it does is that the maxBrightness should correspond with setting the brightness of the device to 100%. So if I would want a max brightness of 75% (which would be 750 out of 1000), I should be able to set maxBrightness to 750 and then when I slide to 100%, this would then send out a value of 750.

I did set the maxBrightness for the device in the IoT platform, but it gets overridden.

The other plugin I tried didn't recognise the Hombli devices so I'm happy with everything this plugin already does. And I'll await your further developments, thank you for putting in the work!

[20221023 23.08 device instruction set.txt](https://github.com/0x5e/homebridge-tuya-platform/files/9847561/20221023.23.08.device.instruction.set.txt)
